nixos/tests/udisks2.nix

```diff
@@ -30,32 +30,41 @@ in
 
   testScript =
     ''
-      my $stick = $machine->stateDir . "/usbstick.img";
-      system("xz -d < ${stick} > $stick") == 0 or die;
+      import lzma
 
-      $machine->succeed("udisksctl info -b /dev/vda >&2");
-      $machine->fail("udisksctl info -b /dev/sda1");
+      with lzma.open(
```

Comment:
Why space this around, `"${stick}"` is much shorter than the line below.

Reply:
black fails on line length, as this becomes a nix store path so it demands I format it this way. We should probably make that more relaxed. Similarly I've had to format ` makeInstalledTest` testscript injection specifically with concatenating multi-line strings so it doesn't error out on it not being pretty.